app/build.gradle

```diff
@@ -1,13 +1,12 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion "25.0.3"
```

Comment:
why did `buildToolsVersion` go away?

Reply:
Gradle was giving me this warning so I removed the line:
```
The specified Android SDK Build Tools version (25.0.3) is ignored, as it is below the minimum supported version (26.0.2) for Android Gradle Plugin 3.0.1.
Android SDK Build Tools 26.0.2 will be used.
To suppress this warning, remove "buildToolsVersion '25.0.3'" from your build.gradle file, as each version of the Android Gradle Plugin now has a default version of the build tools.
```

Reply:
Oh interesting. Didn't know they started applying default versions. Ok makes sense then.

Reply:
I think I will replace `buildToolsVersion "25.0.3"` with `buildToolsVersion "26.0.2"` instead of removing it.